mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 site_name: django-user-management
 pages:
-- [index.md, Home]
+- [README.md, Home]
```

Comment:
Not sure about this `index.md` is a real file in https://github.com/incuna/django-user-management/blob/master/docs/index.md
It's the entry point for the docs